pyperf/_psutil_memory.py

```diff
@@ -0,0 +1,44 @@
+try:
+    import os
+    import psutil
+    import threading
+    import time
+except ImportError:
+    raise ImportError('psutil is not installed')
```

Comment:
Why not pass through the error on "import psutil"?

You may update/clarify when psutil is needed at:
https://pyperf.readthedocs.io/en/latest/run_benchmark.html#install-pyperf

Reply:
Done!